website/docs/r/private_endpoint.html.markdown

```diff
@@ -151,6 +151,12 @@ The following attributes are exported:
 
 * `id` - The ID of the Private Endpoint.
 
+## Example HCL Configurations
+
+* How to connect a `Private Endpoint` to a [Cosmos MongoDB](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples/private_endpoint/cosmosDb)
+* How to connect a `Private Endpoint` to a [PostgreSQL Server](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples/private_endpoint/postgreSql)
+* How to connect a `Private Endpoint` to a [Private Link Service](https://github.com/terraform-providers/terraform-provider-azurerm/tree/master/examples/private_endpoint/privateLinkService)
```

Comment:
since the rest of these examples are using `snake-case` could we fix the paths for these to match? (e.g. `cosmos-db`, `postgresql`, `private-link-service`)

Reply:
yep yep, sure... np 😄